app/controllers/admin_applications_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AdminApplicationsController < ApplicationController
   def show
-    @admin_application = Application.find(params[:app_id])
+    @application = Application.find(params[:app_id])
```

Comment:
Good call on changing this to make the partial actually work. I never thought to change the name as its controller specific.